env.ts

```diff
@@ -58,6 +58,10 @@ const envSchema = {
     desc:       "The URL of the search API.",
     devDefault: "https://discussions-rc.odl.mit.edu/api/v0/search/"
   }),
+  SENTRY_ENV: envalid.str({
+    desc:       "The environment for Sentry",
+    devDefault: "development"
```

Comment:
@fakhar-ud-din re
> Hey @ChristopherChudzicki
> We have merged https://github.com/mitodl/ocw-studio/pull/1725
> Should we also merge this now?

Should be good to merge. See previous note (https://github.com/mitodl/ocw-hugo-themes/pull/1024/files#r1108691183) ... ocw-studio should be deployed first (when it goes to prod; I guess it's already on rc).

Reply:
I guess we should wait till monday then, before releasing https://github.com/mitodl/ocw-studio/pull/1725 to production.

Reply:
https://github.com/mitodl/ocw-studio/pull/1738
this has been merged in the production. Should we also proceed with merging this PR?

Reply:
Should be good to merge.